packages/components/src/private-apis.ts

```diff
@@ -61,4 +62,5 @@ lock( privateApis, {
 	DropdownSubMenuTriggerV2,
 	ProgressBar,
 	Theme,
+	Tabs,
```

Comment:
My nitpicky brain is forcing me to suggest keeping the pre-existing almost-alphabetical sorting 😅 

```suggestion
	Tabs,
	Theme,
```

Reply:
Now I can't unsee it. 😆

Reply:
The struggle!